documentation/modules/auxiliary/admin/kerberos/forge_kerberos_ticket.md

```diff
@@ -0,0 +1,38 @@
+# Kerberos Ticket Forging (Golden/Silver tickets)
+
+The `forge_kerberos_ticket` module allows the forging of a golden or silver ticket.
+
+## Vulnerable Application
+
+Any system leveraging kerberos as a means of authentication e.g. Active Directory, MSSQL
+
+## Pre-Verification steps
+1. Obtain your targets DOMAIN via your favorite method: e.g.
+    `nmap <TARGET_IP>`
+2. Next retrieve the DOMAIN_SID: e.g.
+    `mimikatz # sekurlsa::logonpasswords`
+3. Finally get the NTHASH of the service account you wish to target: e.g.
+    `mimikatz # sekurlsa::logonpasswords` # same command as before, shows you both values
```

Comment:
Can we also mention that these values can be obtained using the `windows_secrets_dump` module.